packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1127,6 +1123,10 @@ mixin WidgetInspectorService {
         return null;
       },
     );
+    registerServiceExtension(
```

Comment:
The service extensions added in this file are copied / moved from https://github.com/flutter/devtools/blob/master/packages/devtools_app/assets/scripts/inspector_polyfill_script.dart.